eng/pipelines/templates/jobs/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,44 @@
+pr: none
```

Comment:
Since this is an entry point yml it should be under the root folder and not under templates.